README.md

```diff
@@ -153,6 +153,40 @@ public function render($request, Throwable $exception)
 }
 ```
 
+### Handling Exception Manually
+
+If you want to handle all exceptions manually, please not that you must convert `ApiResponse` to `JsonResponse` by call
+`toResponse` method explicitly. See this example:
+
```

Comment:
i'll rework some of the wording. but thank you @axmad386 for laying out the base.